[error-tracking] Remove entry span fingerprinting limitation (#26979)
DataDog/documentation: content/en/tracing/error_tracking/_index.md

## Use span attributes to track error spans

> The Datadog tracers collect errors through integrations and the manual instrumentation of your backend services' source code.  Error spans within a trace are processed by Error Tracking if they are service top-level errors, whether or not they are located in service entry spans. This span must also contain the `error.stack`, `error.message`, and `error.type` [span attributes][1] to be tracked.

Reviewer: ```suggestion
The Datadog tracers collect errors through integrations and the manual instrumentation of your backend services' source code. Error Tracking processes error spans within a trace if they occur at the service level, regardless of whether they are located in service entry spans. This span must also contain the `error.stack`, `error.message`, and `error.type` [span attributes][1] to be tracked.
```
I'm not 100% sure what is meant by "service top-level errors". Could you please help provide some clarification so we can make this clear? Thanks!
